scaletest/terraform/gcp_cluster.tf

```diff
@@ -18,7 +18,7 @@ resource "google_container_cluster" "primary" {
 
   }
   release_channel {
-    channel = "STABLE"
+    channel = "UNSPECIFIED"
```

Comment:
nit: maybe leave a comment here? somebody may not figure it out in the future that this is on purpose.